### PR TITLE
chore(ci): disable LFS for suite-web build + tests

### PR DESCRIPTION
**Pull request by Nodonisko**
Removed `lfs` from CI flows where it's not needed. Specifically suite-web e2e did lfs pull in every job in matrix which is a lot. Also added some filter to specific job to fetch only very needed files for connect.

## Related Issue

Resolve

### CODE REVIEW

**Review by karliatto — APPROVED** [review]
Makes sense since we probably no not need all data in those tests.

**Review by vdovhanych — APPROVED** [review]
Should work

File: .github/workflows/connect-dev-release-test.yml [hunk 1]
```diff
@@ -39,9 +39,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          lfs: true
           submodules: recursive
 
+      # Pull only files needed for connect to save LFS bandwidth
+      - name: "Pull LFS files for connect"
+        run: git lfs pull --include "packages/connect-common/files/**/*"
```
Comment (Nodonisko) [inline .github/workflows/connect-dev-release-test.yml:46]: Is this even necessary? Are firmwares bundled into web builds or only to desktop builds? @mroz22

Comment (Nodonisko) [reply .github/workflows/connect-dev-release-test.yml:46]: @karliatto

Comment (Nodonisko) [reply .github/workflows/connect-dev-release-test.yml:46]: It seem's it's included in upload log from AWS so I guess it's needed.

**Pull request merged**